packages/html/src/toolbar/tests/toolbar-sizes.tsx

```diff
@@ -11,7 +11,7 @@ export default () =>(
 
             <span className="k-colspan-all k-col-span-full">Small</span>
             <section className="k-colspan-all k-col-span-full">
-                <Toolbar size="small" resizable>
+                <ToolbarAdaptive size="small">
```

Comment:
Please simplify this test and show all sizes from the Toolbar options. Like here:
https://github.com/telerik/kendo-themes/blob/develop/packages/html/src/pager/tests/pager-sizes.tsx#L7